common/src/chain/block/timestamp.rs

```diff
@@ -15,12 +15,13 @@
 
 use parity_scale_codec::{Decode, Encode};
 use std::time::Duration;
-use thiserror::Error;
+
+pub type BlockTimestampInternalType = u64;
```

Comment:
Why introduce a new type?

Reply:
Just for consistency in the file... in case we think of changing it, it'll change everything automatically. I hate repeating the same thing multiple times.

Reply:
By the way, we have slightly differently named type aliases in other places. For example, `HeightIntType` or `DistanceIntType`. Should this also be `BlockTimestampIntType`? Or do we want to emphasize that it is an internal type?